sycl/test-e2e/NewOffloadDriver/aot-cpu.cpp

```diff
@@ -6,5 +6,5 @@
 // RUN: %{run} %t.out
 
 // Test -O0 with `--offload-new-driver`
-// RUN: %clangxx -O0 -fsycl -fsycl-targets=spir64-x86_64 %S/Inputs/aot.cpp
+// RUN: %clangxx %no_opt -fsycl -fsycl-targets=spir64-x86_64 %S/Inputs/aot.cpp
```

Comment:
I think that `%O0` would be a better name for this substitution. 

The thing is that we have two different methods of disabling optimizations and `no_opt` is a bit confusing:
- We have `-O0` which disables both optimizations made by SYCL compiler and also records some metadata into device images which later leads to backend optimizations also being disabled
- We also have `-fno-sycl-early-optimizations`, which *only* disables optimizations made by SYCL compiler, but preserves optimizations made by device compilers later

Even though optimization pipeline is very similar (if not the same) for both options, resulting LLVM IR will differ between them. `-O0` will have `optnone` attached to functions, whilst `-fno-sycl-early-optimizations` won't result in such attribute